### ISSUE TITLE
Fails to serve svg files

### ISSUE THREAD

**Issue by Stefan-Z-Camilleri** [simonh1000/angular-http-server#5]
Hi, this project is great, except that it fails to serve svg files.

This works fine with `http-server`

The problem is the mime mapping. `http-server` returns svg as `image/svg+xml; charset=utf-8` whilst `angular-http-server` returns them as `text/svg`

**Comment by simonh1000** [comment]
Thanks for this - looks like I forgot to publish the latest update to the code that had a fix for that already. Try updating now (to 1.3.1) and let me know how you get on

**Comment by Stefan-Z-Camilleri** [comment]
Yep, just saw Chris's commit, that should fix it.  Thank you @simonh1000 , I will let you know if there are any other problems.

Keep up the good work, and thank you for this.

**Comment by Stefan-Z-Camilleri** [comment]
Just tested this, it looks like there's a missing dependency on `minimist`.

Once I manually add this, everything works just fine.

**Comment by greenygh0st** [comment]
Are you getting an error about the dep? I just pulled the new version down and everything is working as expected.

```
redacted:~ dale$ angular-http-server 
Listening on 8080
```

**Comment by greenygh0st** [comment]
If you are installing it as a part of your app with "npm install" wipe the node_modules folder and try again. If it's global uninstall and re-install.

See if either of those two work for you.

**Comment by simonh1000** [comment]
This does feel like an issue changing between versions as minimist is in package.json https://github.com/simonh1000/angular-http-server/blob/master/package.json#L31

**Comment by Stefan-Z-Camilleri** [comment]
Sorry for the late reply... been busy :)

TLDR; you're correct, this is an issue with upgrade... since I used upgrade.

In the meantime I had actually checked your package.json and saw the dependency was correctly there.

Currently installation of this is part of a Dockerfile, and in fact, upon re-building (i.e. clean node_modules) everything worked fine.

I think you can close this off @simonh1000 since the pending annoyance seems more npm related.
